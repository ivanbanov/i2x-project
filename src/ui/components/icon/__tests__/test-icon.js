import React from 'react';
import renderer from 'react-test-renderer';
import Icon from '../';

jest.mock('../styles.styl', () => ({}));

describe('Grid', () => {
  it('should render properly', () => {
    const component = renderer.create(
      <Icon name="foo" className="bar" />
    );

    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
