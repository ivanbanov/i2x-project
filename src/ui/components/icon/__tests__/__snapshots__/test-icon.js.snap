// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid should render properly 1`] = `
<svg
  className="bar"
>
  <use
    xlinkHref="[object Object]#foo"
  />
</svg>
`;
